src/RecyclableMemoryStreamManager.cs

```diff
@@ -103,11 +106,12 @@ public RecyclableMemoryStreamManager()
         /// Initializes the memory manager with the given block requiredSize.
         /// </summary>
         /// <param name="blockSize">Size of each block that is pooled. Must be > 0.</param>
-        /// <param name="largeBufferMultiple">Each large buffer will be a multiple of this value.</param>
+        /// <param name="largeBufferMultiple">Each large buffer will be a multiple/exponential of this value.</param>
         /// <param name="maximumBufferSize">Buffers larger than this are not pooled</param>
+        /// <param name="useExponentialLargeBuffer">Switch to exponential large buffer allocation strategy</param>
         /// <exception cref="ArgumentOutOfRangeException">blockSize is not a positive number, or largeBufferMultiple is not a positive number, or maximumBufferSize is less than blockSize.</exception>
-        /// <exception cref="ArgumentException">maximumBufferSize is not a multiple of largeBufferMultiple</exception>
-        public RecyclableMemoryStreamManager(int blockSize, int largeBufferMultiple, int maximumBufferSize)
+        /// <exception cref="ArgumentException">maximumBufferSize is not a multiple/exponential of largeBufferMultiple</exception>
+        public RecyclableMemoryStreamManager(int blockSize, int largeBufferMultiple, int maximumBufferSize, bool useExponentialLargeBuffer = false)
```

Comment:
This will cause a break if someone replaces the DLL without rebuilding. Add another constructor instead.